### PR TITLE
Rename Cozy Files to Cozy Drive

### PR DESCRIPTION
**Pull request by kosssi**
On this repository:
- [x] Rename cozy-files-v3 to cozy-drive
- [x] Rename Cozy Files to Cozy Drive
- [x] Update mobile id
- [x] Update translation on project

Update service:
- [ ] Update android app id
- [ ] Update iOS app id
- [ ] Rename Github repository
- [ ] Rename Transifex project
- [ ] Update translation on Transifex
- [ ] Update Travis

### CODE REVIEW

File: manifest.webapp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Files",
+  "name": "Cozy Drive",
```
Comment (enguerran) [inline manifest.webapp]: I would do s/Files/Drive/ as this manifest is used by the cozy-stack (in which the word Cozy is obvious)

File: src/locales/en.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
 {
   "nav": {
-    "item_files": "Files",
+    "item_files": "Drive",
```
Comment (enguerran) [inline src/locales/en.json]: I would keep *Files* in that context.
But if you change the value, consider changing the key as well.

File: src/locales/en.json [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
     "item_recent": "Recent",
     "item_shared": "Shared by me",
     "item_activity": "Activity",
     "item_trash": "Trash",
     "item_settings": "Settings"
   },
   "breadcrumb": {
-    "title_files": "Files",
+    "title_files": "Drive",
```
Comment (enguerran) [inline src/locales/en.json]: I would keep *Files* in that context.
But if you change the value, consider changing the key as well.
And to depend less on the translation, maybe `breadcrumb.title` is explicit enough.

Comment (jsilvestre) [inline src/locales/en.json]: We chose to go for "Drive" here to match what is done by Google, but @pierrevdk will have the last word anyway

File: src/locales/en.json (hunk 2 above)
Comment (jsilvestre) [inline src/locales/en.json]: Same comment than for the breadcrumb, @pierrevdk will decide what they want, but as for now we copy what Google does (= using Drive everywhere).

**Pull request merged**